modules/menu/css/stories.tsx

```diff
@@ -23,41 +17,17 @@ storiesOf('CSS/Menu', module)
         <div className="wdc-menu">
           <ul>
             <li>
-              <a href="#">Item 1</a>
-            </li>
-            <li>Item 2</li>
-          </ul>
-        </div>
-        <div className="wdc-menu wdc-menu-right">
-          <ul>
-            <li>
-              <a href="#">Item 1</a>
+              <a href="#">First Item</a>
             </li>
-            <li>Item 2</li>
-          </ul>
-        </div>
-        <div className="wdc-menu wdc-menu-left">
-          <ul>
-            <li>
-              <a href="#">Item 1</a>
-            </li>
-            <li>Item 2</li>
-          </ul>
-        </div>
-        <div className="wdc-menu wdc-menu-top">
-          <ul>
-            <li>
-              <a href="#">Item 1</a>
-            </li>
-            <li>Item 2</li>
-          </ul>
-        </div>
-        <div className="wdc-menu wdc-menu-bottom">
-          <ul>
+            <li>Second Item with really really really long label</li>
+            <li className="is-disabled">Third Item (disabled)</li>
```

Comment:
Is the `is-disabled` class properly named?

Reply:
We generally prefix our classes. Can we call this `wdc-menu-item-disabled`?

Reply:
Done.